### PR TITLE
add owners file

### PR DESCRIPTION
**Pull request by nlopezgi**
https://github.com/bazelbuild/rules_docker/issues/784

### CODE REVIEW

**Comment by nlopezgi** [comment]
fyi @fejta

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# See Config struct in https://github.com/kubernetes/test-infra/blob/master/prow/repoowners/repoowners.go
+
+# Reviewers may be auto-assigned by blunderbuss
+reviewers:
```
Comment (fejta) [inline OWNERS]: When the reviewers and approvers section is the same, you can just have the approvers section (we'll read reviewers from the approvers section in this scenario)

**Comment by fejta** [comment]
@smukherj1 any chance you can review this today so we can merge it?

**Comment by fejta** [comment]
FYI http://gubernator.k8s.io/pr will give you a handy list of PRs you need to approve

**Pull request merged**